src/cgame/cg_event.cpp

```diff
@@ -134,7 +134,7 @@ static void CG_Obituary( entityState_t *ent )
 	ci = &cgs.clientinfo[ target ];
 	gender = ci->gender;
 
-	if ( attacker < 0 || attacker >= MAX_CLIENTS )
```

Comment:
This bounds checking shouldn't be removed

Reply:
I'll fix that.

Reply:
Fixed in e036bf3c1cb449fd5fdb6be3b247cea403f6775a